aten/src/ATen/gen.py

```diff
@@ -43,6 +43,18 @@
     '--rocm',
     action='store_true',
     help='reinterpret CUDA as ROCm/HIP and adjust filepaths accordingly')
+parser.add_argument(
+    '--op_registration_whitelist',
+    nargs='*',
+    help='filter op registrations by the whitelist (if set)')
+parser.add_argument(
+    '--type_whitelist',
+    nargs='*',
+    help='filter types (CPUType, CUDAType, ...) by the whitelist (if set)')
```

Comment:
Noooo more public references to CPUType XD (no action needed)

Reply:
strongly agree.  CPUType is an implementation detail.  What do you actually need to whitelist?  If it's just stuff from native_functions, just use the keys from there.

Reply:
> strongly agree. CPUType is an implementation detail. What do you actually need to whitelist? If it's just stuff from native_functions, just use the keys from there.

To clarify, do you mean using CPU / CUDA / MkldnnCPU / QuantizedCPU - dispatch keys used in native_functions.yaml? I'll update.

Reply:
Yep!